docs/source/custom-graphs.rst

```diff
@@ -55,6 +55,15 @@ analyze pipeline:
    from dask.multiprocessing import get
    get(dsk, 'store')  # executes in parallel
 
+In the above example, while multiprocessing, if a notebook is not being 
+used to run, the ``get(dsk, 'store')`` command needs to be run inside an 
+``if __name__ == '__main__':`` block as shown below:
+
+.. code-block:: python
+
+   if __name__ == '__main__':
+      get(dsk, "store")
```

Comment:
```suggestion
If the above example is not being run inside a notebook, the ``get(dsk, 'store')``
command should be placed inside an ``if __name__ == '__main__':`` block:

.. code-block:: python

   if __name__ == '__main__':
       get(dsk, "store")
       
For an explanation of why ``if __name__ == '__main__'`` is necessary, see the
standard `multiprocessing guidelines <https://docs.python.org/3/library/multiprocessing.html#programming-guidelines>`_
for Python.
```

What do you think about this revision? (Your version is good - just adding slightly more context)

Reply:
Looks great!!